corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -70,8 +70,7 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
                             unique_form_id=form.unique_id)
 
     vellum_plugins = ["modeliteration", "itemset"]
-    if (toggles.VELLUM_TRANSACTION_QUESTION_TYPES.enabled(domain) or
-            toggles.COMMTRACK.enabled(domain)):
+    if (toggles.COMMTRACK.enabled(domain)):
```

Comment:
@emord @millerdev confirming that there wasn't any other magic in formbuilder that would be affected by this